### PR TITLE
Add doukutsu-rs

### PR DESCRIPTION
**Pull request by alula**

### CODE REVIEW

File: games/d.yaml [hunk 1]
```diff
@@ -642,6 +642,21 @@
...
   url: https://dopewars.sourceforge.io/
 
+- name: doukutsu-rs
+  images:
+  - https://i.imgur.com/eZ0V5rK.png
+  lang:
+  - Rust
+  license:
+  - MIT
+  development: active
```
Comment (cxong) [inline games/d.yaml:652]:
```suggestion
  development: active
  status: unplayable
```

File: games/d.yaml [hunk 2]
```diff
@@ -642,6 +642,21 @@
   updated: 2013-05-27
   url: https://dopewars.sourceforge.io/
 
+- name: doukutsu-rs
+  images:
+  - https://i.imgur.com/eZ0V5rK.png
+  lang:
+  - Rust
+  license:
+  - MIT
```
Comment (cxong) [inline games/d.yaml:651]:
```suggestion
  - MIT
  content: free
```

File: games/d.yaml [hunk 3]
```diff
@@ -642,6 +642,21 @@
...
+  development: active
+  originals:
+  - Cave Story
+  repo: https://github.com/doukutsu-rs/doukutsu-rs
+  type: remake
+  updated: 2020-10-30
+  url: https://github.com/doukutsu-rs/doukutsu-rs
```
Comment (cxong) [inline games/d.yaml:658]:
```suggestion
```
Redundant with repo

File: games/d.yaml (hunk 1 above)
Comment (alula) [reply games/d.yaml:652]: is there a status for "playable but incomplete" or it counts as "unplayable"?

**Comment by alula** [comment]
applied

File: games/d.yaml (hunk 1 above)
Comment (cxong) [reply games/d.yaml:653]: https://github.com/opengaming/osgameclones/blob/906d30eca4e50596bd615053f867b57cfc2c191a/schema/games.yaml#L38

semi-playable

**Comment by alula** [comment]
can me merged now ig

**Review by cxong — APPROVED** [review]

**Pull request merged**